src/cider/nrepl/middleware/debug.clj

```diff
@@ -477,13 +477,12 @@ this map (identified by a key), and will `dissoc` it afterwards."}
                                  ;; Passing clojure.lang.Namespace object
                                  ;; as :original-ns breaks nREPL in bewildering
                                  ;; ways.
+                                 ;; NOTE: column numbers in the response map
```

Comment:
Probably we should also mention this in the debugger docs.

Reply:
Do you mean the docs in this repo or Cider's?

Reply:
CIDER's. I think we don't have any debugger documentation elsewhere, although we should change this at some point.